### PR TITLE
Update release notes for #27043

### PR DESCRIPTION
**Pull request by reinsch82**
Fixes 
 - #27043

### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -87,6 +87,12 @@ The information comes from `@since` tags in the Javadoc, but which weren't displ
 Using the new `--no-comments` option allows generating projects that contain only code without clarifying comments.
 The resulting build files and source files are smaller and less noisy.
 
+#### Emitting separate problems for each issue found in a TOML catalog file
+
+When a TOML catalog files are used with invalid TOML syntax every distinct problem is emitted separately via the Problems API.
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:92]:
```suggestion
When TOML catalog files with invalid TOML syntax are used, each distinct problem is reported separately using the Problems API.
```

Would it be possible to include a screenshot or a snapshot of the report?

**Review by lkasso — CHANGES_REQUESTED** [review]
It looks like it's in the middle of another section of the release notes, can you move it @reinsch82?

File: platforms/documentation/docs/src/docs/release/notes.md (hunk 1 above)
Comment (reinsch82) [reply platforms/documentation/docs/src/docs/release/notes.md:92]: It's just an even we send, what it looks like in a report entirely depends on the consumer e.g. the IDEs.

Comment (lkasso) [reply platforms/documentation/docs/src/docs/release/notes.md]: Sounds good, no screenshot, just the grammar update and we are good to go.

**Comment by donat** [comment]
@bot-gradle merge

**Comment by ljacomet** [comment]
@bot-gradle cancel

**Pull request closed without merging**